tests/autoria/test_select.py

```diff
@@ -0,0 +1,17 @@
+from autoria.api import select_item
+
+
+class TestSelect:
+    """Tests for ``select_item`` function."""
+
+    def test_select_exact(self):
```

Comment:
You need to add more test for select_item (better as additional PR) to check wild-card matching.